src/main/java/teammates/common/datatransfer/CourseAttributes.java

```diff
@@ -28,29 +28,37 @@ public CourseAttributes() {
     }
 
     public CourseAttributes(String courseId, String name) {
-        this.id = Sanitizer.sanitizeTitle(courseId);
+        id = Sanitizer.sanitizeTitle(courseId);
         this.name = Sanitizer.sanitizeTitle(name);
-        this.isArchived = false;
+        isArchived = false;
     }
     
     public CourseAttributes(String courseId, String name, boolean archiveStatus) {
-        this.id = Sanitizer.sanitizeTitle(courseId);
+        id = Sanitizer.sanitizeTitle(courseId);
         this.name = Sanitizer.sanitizeTitle(name);
-        this.isArchived = archiveStatus;
+        isArchived = archiveStatus;
```

Comment:
In cases like this, it may be better to retain the `this.` for consistency's sake.
Unlike in getters that return a single variable in a single line, these constructors dealing with more variables, all with class-scope can benefit from a shared identity by starting with `this.`.

Also, you still have redundant `this.`s. Take a look at the PR's diff view and scroll down a little.